File: AndroidShimmerDemo/app/src/main/java/com/example/shakil/androidshimmerdemo/Retrofit/IMyApi.java
package com.example.shakil.androidshimmerdemo.Retrofit;

import com.example.shakil.androidshimmerdemo.Model.Model;

import java.util.List;

import io.reactivex.Observable;
import retrofit2.http.GET;

public interface IMyApi {
    @GET("photos")
    Observable<List<Model>> getDate();
}

File: KotlinShimmerDemo/app/src/main/java/com/example/shakil/kotlinshimmerdemo/Retrofit/IMyAPI.kt
package com.example.shakil.kotlinshimmerdemo.Retrofit

import com.example.shakil.kotlinshimmerdemo.Model.Model
import io.reactivex.Observable
import retrofit2.http.GET

interface IMyAPI {
    @get:GET("photos")
    val data: Observable<List<Model>>
}